app/src/org/commcare/dalvik/dialogs/AlertDialogFactory.java

```diff
@@ -81,12 +82,11 @@ public void onClick(DialogInterface dialog, int which) {
     }
 
     public void showDialog() {
-        dialog.setView(this.view);
```

Comment:
@phillipm There's a reason this call needs to be here -- there are methods that can be called on an instance of AlertDialogFactory in between its construction and calling showDialog() that update the view.